src/api/helpers/decrypt-financial-uk/index.test.ts

```diff
@@ -0,0 +1,35 @@
+import decryptFinancialUk from '.';
+import mockApplication from '../../test-mocks/mock-application';
+import decryptData from '../decrypt';
+
+const {
+  nominatedLossPayee: { financialUk },
+} = mockApplication;
+
+const decryptSpyResponse = '123456';
+
+describe('api/helpers/decrypt-financial-uk', () => {
+  jest.mock('../decrypt');
+
+  let decryptSpy = jest.fn();
+
+  beforeEach(async () => {
+    jest.resetAllMocks();
+
+    decryptSpy = jest.fn(() => decryptSpyResponse);
+
+    decryptData.decrypt = decryptSpy;
+  });
+
+  it('should return decrypted sortCode and accountNumber', () => {
+    const result = decryptFinancialUk(financialUk);
```

Comment:
Can you please add an assertion where the input is `undefined`, `{}` and with an exception please.

Reply:
I think TypeScript will automatically handle this. I.e TS should throw an error (and linting issue) if an empty object or undefined is passed into the function. 

Unless the `ApplicationFinancialUk` type has optional properties, in which case we need to test empty object scenario.